websites/R/Roblox/presence.ts

```diff
@@ -1,252 +1,713 @@
 const presence = new Presence({
 		clientId: "612416330003382314"
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingStamp = Math.floor(Date.now() / 1000),
+	dfLgImage = "https://i.imgur.com/76AO77L.png";
 let profileName,
+	profileId: string,
+	profileImg: string,
 	profileTabs,
-	profileAvatar,
 	messageTab,
 	friendsTab,
 	inventoryTab,
 	groupName,
-	groupImage,
 	groupTab,
-	gameTab;
+	groupImage,
+	gameId: string,
+	gameImage: string,
+	gameTab,
+	localizationTab: HTMLSpanElement,
+	localizationGameName,
+	transactionsTab,
+	dfPrevTopic: string,
+	dfTopicName = "[Loading...]",
+	dfTopicTime: number,
+	talentUserData: [string, string] = ["0", "0"];
 
 presence.on("UpdateData", async () => {
-	const presenceData: PresenceData = {
+	const buttons = await presence.getSetting("buttons"),
+		imagesEnabled = await presence.getSetting<boolean>("images"),
+		presenceData: PresenceData = {
 			details: "Unknown page",
-			largeImageKey: "lg",
-			startTimestamp: browsingTimestamp
```

Comment:
Why have you removed this? This would remove the unnecessary clutter of repeating `presenceData.startTimestamp = browsingTimestamp` through the presence